templates/zshrc.j2

```diff
@@ -27,10 +27,16 @@ ZSH_THEME="{{ oh_my_zsh.theme | default(oh_my_zsh_theme) }}"
 # Case-sensitive completion must be off. _ and - will be interchangeable.
 # HYPHEN_INSENSITIVE="true"
 
-# Uncomment one of the following lines to change the auto-update behavior
-zstyle ':omz:update' mode disabled  # disable automatic updates
-# zstyle ':omz:update' mode auto      # update automatically without asking
-# zstyle ':omz:update' mode reminder  # just remind me to update when it's time
+{% if 'auto' == oh_my_zsh.update_mode %}
```

Comment:
I expect this will fail if `oh_my_zsh.update_mode` is undefined, `oh_my_zsh_update_mode` is documented but not used. Also, can you update the Molecule tests to include this setting.